packages/orchestrator/internal/template/build/layer/create_sandbox.go

```diff
@@ -65,10 +78,8 @@ func (cs *CreateSandbox) Sandbox(
 	}
 
 	// In case of a new sandbox, base template ID is now used as the potentially exported template base ID.
-	sbx, err := sandbox.CreateSandbox(
+	sbx, err := cs.sandboxFactory.CreateSandbox(
 		ctx,
-		layerExecutor.networkPool,
-		layerExecutor.devicePool,
```

Comment:
I think layer executor doesn't need those anymore

Reply:
Good catch! Removed them and a bunch of other unused fields.